src/stan/io/stan_csv_reader.hpp

```diff
@@ -256,14 +256,14 @@ namespace stan {
             break;
 
           if (comment_line) {
-            if (line.find("(Warm-up)") != std::string::npos) {
-              int left = 17;
-              int right = line.find(" seconds");
+            if (line.find("Warmup") != std::string::npos) {
```

Comment:
Bob, you�re looking at the wrong branch.  This is how it�s currently implemented
in develop � in feature/issue-1361-writer_callback this is done much more robustly.

On Aug 19, 2015, at 10:56 AM, Bob Carpenter <notifications@github.com> wrote:

> In src/stan/io/stan_csv_reader.hpp:
> 
> > @@ -256,14 +256,14 @@ namespace stan {
> >              break;
> >  
> >            if (comment_line) {
> > -            if (line.find("(Warm-up)") != std::string::npos) {
> > -              int left = 17;
> > -              int right = line.find(" seconds");
> > +            if (line.find("Warmup") != std::string::npos) {
> 
> What in the world is there a hard coded 17 in here for? Isn't that going to make things rather brittle?
> �
> �
> Reply to this email directly or view it on GitHub.
>